game.py

```diff
@@ -80,33 +86,48 @@ def game_main():
     ]
     
     # These are the only input values that can be used in the game currently
-    valid_inputs = ["rest", "mine", "fish", "combat"]
+    valid_inputs = ["rest", "mine", "fish", "combat", "save-test", "actions"]
 
     while True:  # Keeps the game going after every action (always truthy unless error)
 
-        get_input = input("What Action Are You Doing? (rest, mine, fish, combat)\n\n")
-        os.system('clear')  # Clean the terminal again!
+        get_input = input("What action do you want to perform? (Type: 'actions' to display actions.)\n\n")
         
-        if player.current_energy > 0:  # If player_energy is greater than 0, keep going
+        clear_term()
         
-            if get_input in valid_inputs:  # If input is in valid_inputs, keep going
-            
-                if get_input == "rest": 
+        if player.current_energy > 0:
+            if get_input in valid_inputs:
+                if get_input == "rest":
 
-                    # Display a random rest message
                     message_rest = random_rest_messages[(rand.randint(0, 4))]
                     print(message_rest, '\n')
 
                     player.rest()
+
+                    get_cur_stats(skills_list, player)
+
+                elif get_input == "save-test":
```

Comment:
save-test gets all the relevant data from class.Player and passes it over to the save_player_data function stored in savegame.py